README.md

```diff
@@ -69,25 +32,30 @@ Much of the philosophy behind Bedrock is inspired by the [Twelve-Factor App](htt
 - Autoloader for mu-plugins (use regular plugins as mu-plugins)
 - Enhanced security (separated web root and secure passwords with [wp-password-bcrypt](https://github.com/roots/wp-password-bcrypt))
 
-## Requirements
+## How to use this project
+There are two main ways to interact with this project template. **Using the Pantheon-maintained WordPress Composer Managed upstream** or **forking this repository to create a custom upstream.**
+
+### Using the Pantheon WordPress Composer Managed upstream
+
+1. Use Terminus to create a site from the Pantheon upstream:
+```
+terminus site:create --org ORG --region REGION -- site_name> <label> "WordPress Composer Managed"
+```
+1. In the Dev environment, click **Visit Development Site** and follow the prompts to complete the CMS installation.
+2. [Clone the site locally](https://pantheon.io/docs/local-development#get-the-code) and run `composer install`.
+
+### Fork this repository to create a custom upstream
+1. Fork this repository into your own GitHub profile.
+2. [Add a new Custom Upstream](https://pantheon.io/docs/guides/custom-upstream/create-custom-upstream#connect-repository-to-pantheon) on the Pantheon Dashboard.
+3. Create a new WordPress site from the Upstream.
 
-- PHP >= 7.4
-- Composer - [Install](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx)
+**Note:** It's highly recommended that you use the Pantheon-maintained upstream in favor of creating and managing a custom upstream so you can be sure to receive the latest updates. Managing your own custom upstream means that you assume ownership of the upstream and all changes made to it and assumes that you will manage all updates to the upstream.
```

Comment:
What do you think about moving this line under the heading "### Fork this repository to create a custom upstream" and changing the heading to something indicating this section is optional?